requirements/env_common.yml

```diff
@@ -23,7 +23,7 @@ dependencies:
     - pymatgen
     - simdna
     - transformers==4.34.1
-    - tokenizers
-    - xgboost
+    - tokenizers==0.14.1
+    - xgboost==1.4.0
```

Comment:
We are explicitly pinning to an old xgboost version here (latest is 2.0.1) which we don't want to do. We need to instead update the deepchem xgboost support to the latest version

Reply:
sure, ill make the code changes